tests/test_du.rs

```diff
@@ -0,0 +1,124 @@
+use common::util::*;
+
+static SUB_DIR: &str = "subdir/deeper";
```

Comment:
Can you change these to `const` instead of `static`?  Most of the things in the codebase using `static` just haven't been updated (that used to be the only to have constants).